style.css

```diff
@@ -743,3 +755,803 @@ input::placeholder {
+    position: absolute;
+    width: 1397px;
+    height: 120px;
+    left: 0;
+    top: 0;
+    margin-left: 8px;
+    background-color: #fff;
+  }
+
+  ul {
+    list-style: none;
+    padding: 0;
+    margin: 0;
+  }
+
+  .logo {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 10px 12px;
+    position: absolute;
+    width: 100px;
+    height: 40px;
+    left: 100px;
+    top: 20px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 20px;
+    text-align: center;
+    color: #6070ff;
+  }
+
+  .desktop-nav {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+    gap: 8px;
+    position: absolute;
+    width: 259px;
+    height: 40px;
+    left: 1080px;
+    top: 50px;
+    color: #000;
+  }
+
+  .navbar-tools {
+    width: 63px;
+    height: 20px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    color: #344563;
+  }
+
+  .headline {
+    background-image: url("/images/Header\ bg.svg");
+    background-repeat: no-repeat;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    position: absolute;
+    width: 1397px;
+    height: 800px;
+    gap: 50px;
+    top: 100px;
+    border-radius: 0 0 0 70px;
+    background-color: #fff;
+  }
+
+  #h1 {
+    position: relative;
+    top: 0;
+    width: 600px;
+    height: 120px;
+    padding-left: 170px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    display: flex;
+    align-items: center;
+    letter-spacing: 0.37px;
+    color: #091e42;
+    margin-left: 40px;
+  }
+
+  .paragraph {
+    position: relative;
+
+    /* top: 40px; */
+    padding-left: 200px;
+    width: 920px;
+    height: 98px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #344563;
+  }
+
+  .connect {
+    position: relative;
+    top: -35px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    padding-left: 200px;
+    color: #7f8cff;
+    flex: none;
+    margin-left: 15px;
+  }
+
+  .social-icons {
+    position: relative;
+    top: -50px;
+    left: 201px;
+    width: 170.17px;
+    height: fit-content;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    list-style-type: none;
+    padding-left: 0;
+  }
+
+  .desktop-project-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 60px;
+    gap: 142px;
+    position: absolute;
+    width: 1397px;
+    height: 2600px;
+    left: 0;
+    top: 1000px;
+  }
```

Comment:
- [ ] Please make sure our project cards are centered on the page, as it is in the figma template, currently it is aligned to the right, please see 
![image](https://user-images.githubusercontent.com/97521557/208231840-a8272a00-8f45-42dd-b6f4-d79f510c9de0.png)

As one of the goal is to parse the figma template as close as possible let's try to make the project cards like in the figma template see
![image](https://user-images.githubusercontent.com/97521557/208231919-73b32e77-daa7-478b-bda4-e64200078bc7.png)

I think if you remove the `align-items :center ` property from your `.desktop-project-container ` we can fix this issue.

Reply:
Thank you for the recommendation, changed it as suggested.